scripts/build/Platform/Windows/env_windows.cmd

```diff
@@ -17,8 +17,8 @@ IF NOT %ERRORLEVEL%==0 (
 )
 
 IF NOT "%COMMAND_CWD%"=="" (
-    ECHO [ci_build] Changing CWD to %COMMAND_CWD%
-    CD %COMMAND_CWD%
+    ECHO [ci_build] Changing CWD to !COMMAND_CWD!
+    CD !COMMAND_CWD!
```

Comment:
is it OK if the COMMAND_CWD folder doesn't exist?  if not we could check if the CD command failed

```
IF NOT %ERRORLEVEL%==0 (
    ECHO [ci_build] Failed to change directory to !COMMAND_CWD!
    GOTO :error
)
```

Reply:
+1, will add that too, the job should fail if it doesnt